cql/evaluate-measure-hapi.sh

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env bash
+
+# Usage: ./evaluate-measure.sh -f <query>.cql <server-base>
+
+# Takes a CQL file, creates a Library resource from it, references that from a
+# Measure resource and calls $evaluate-measure on it.
+
+library() {
+cat <<END
+{
+  "resourceType": "Library",
+  "status": "active",
+  "type" : {
+    "coding" : [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/library-type",
+        "code" : "logic-library"
+      }
+    ]
+  },
+  "content": [
+    {
+      "contentType": "text/cql"
+    }
+  ]
+}
+END
+}
+
+measure() {
+cat <<END
+{
+  "resourceType": "Measure",
+  "status": "active",
+  "subjectCodeableConcept": {
+    "coding": [
+      {
+        "system": "http://hl7.org/fhir/resource-types",
+        "code": "Patient"
+      }
+    ]
+  },
+  "scoring": {
+    "coding": [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/measure-scoring",
+        "code": "cohort"
+      }
+    ]
+  },
+  "group": [
+    {
+      "population": [
+        {
+          "code": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+                "code": "initial-population"
+              }
+            ]
+          },
+          "criteria": {
+            "language": "text/cql",
+            "expression": "InInitialPopulation"
+          }
+        }
+      ]
+    }
+  ]
+}
+END
+}
+
+fhir-helpers(){
+  cat <<END
+{
+      "id":"FHIRHelpers",
+      "name":"FHIRHelpers",
+      "version":"4.0.0",
```

Comment:
Is it possible that HAPI ships with a 4.0.1 FHIRHelpers Library? I started with 4.0.0 and never updated but we can clearly use 4.0.1.

Reply:
Seems like they did not include the 4.0.1 FHIRHelpers Library. I modified the t-1-1.cql to use 4.0.1 FHIRHelpers but the Fhir Server throws the error that it cannot resolve the Library.

Reply:
Ok, but we don't need to update the FHIRHelpers on every query. So I would rather provide a script to create the FHIRHelpers once and document that in a README.